internal/convert/convert.go

```diff
@@ -442,6 +457,23 @@ func OperandsToString(operands []model.IExpression) string {
 	return stringOperands.String()
 }
 
+// operandsToStringForTypes returns a string representation of type operands.
+// This is similar to OperandsToString but works on IType slices instead of IExpression slices.
+func operandsToStringForTypes(operands []types.IType) string {
+	var stringOperands strings.Builder
+	for i, operand := range operands {
+		if i > 0 {
```

Comment:
Wouldn't this also add ", " after the last element? I think it would make more sense to invert this and add this after the current operand is added only if we're not currently at the end of the list.